.editorconfig

```diff
@@ -20,7 +20,7 @@ charset = utf-8
 indent_size = 4
 # isort plugin configuration
 known_first_party = invenio_records_permissions
-multi_line_output = 2
+multi_line_output = 5
```

Comment:
Hmm... This comes from cookiecutter-invenio-module. If it should be changed, could you make a PR there? if its a local default I think we should avoid pushing them (maybe add to the .gitignore?)

Reply:
:+1:  I will be making a PR for a couple of changes to that repository. 

The isort plugin configuration in `.editorconfig` is strangely enough being picked up by the isort command being run in tests! I wouldn't have expected that: thought `.editorconfig` was just for editors...

Reply:
I have tried using `__all__` but it ended up being more clunky (at least I thought so). If this change in `multi_line_output` is really not desired, I can go back to `__all__`.

Reply:
I understand the issues and I share the pain. However, personally I would not change that config here until the architects approve the change in the cookiecutter repo. Mainly to avoid inconsistencies across repos. Once its merge we can put a task and change it across repos.